### DISCUSSION TITLE
Issue while importing Jax

### DISCUSSION THREAD

**Discussion by shashilakra** [jax-ml/jax#18080] [Q&A]
Hi All,

Currently I am working on project which has dependency on jax version 0.2.13. But the problem is whenever I am trying to import jax, I am getting below error:

**[libprotobuf ERROR google/protobuf/descriptor_database.cc:641] File already exists in database: tensorflow/compiler/xla/xla_data.proto
[libprotobuf FATAL google/protobuf/descriptor.cc:2021] CHECK failed: GeneratedDatabase()->Add(encoded_file_descriptor, size):
Aborted (core dumped)**

Please note that the project I am working doesn't has any depenedency on tensorflow, even if I installed the tensorflow I am still getting same error.

**Comment by jakevdp** [comment]
This looks like it's coming from the protobuf package. JAX 0.2.13 is quite old – I wonder if other packages in your stack are too new? What version of `protobuf` do you have installed? When JAX 0.2.13 was released, the current release of `protobuf` was v0.3.16.
